src/components/AssetsPage/index.jsx

```diff
@@ -151,11 +151,13 @@ export default class AssetsPage extends React.Component {
     return (
       <div className={styles.assets}>
         <div className={edxBootstrap.container}>
-          <div className={edxBootstrap.row}>
-            <div className={`${edxBootstrap['col-md-3']} ${edxBootstrap['offset-md-9']}`}>
-              <WrappedAssetsSearch />
+          {this.props.searchSettings.enabled &&
```

Comment:
I modeled this conditional rendering from a [reactjs example](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).